test/gen_rmq_consumer_test.exs

```diff
@@ -92,6 +97,80 @@ defmodule GenRMQ.ConsumerTest do
     close_connection_and_channels_after_shutdown_test()
   end
 
+  describe "TestConsumer.ErrorInConsumer" do
+    setup :attach_telemetry_handlers
+
+    setup do
+      Agent.start_link(fn -> MapSet.new() end, name: ErrorInConsumer)
+      with_test_consumer(ErrorInConsumer)
+    end
+
+    test "should invoke the consumer's handle_info callback if error exists",
+         %{consumer: consumer_pid, state: state} = context do
+      clear_mailbox()
+      message = %{"value" => 0}
```

Comment:
Can we make it clearer how this triggers an error in the consumer? Even just a comment would help clarify that this will trigger a division by zero.
Or make it even more explicit in ErrorConsumer with something like `if message.value == 0, do: raise "Error!"`